app/src/main/java/com/keylesspalace/tusky/adapter/StatusBaseViewHolder.java

```diff
@@ -123,8 +129,10 @@ private void setUsername(String name) {
     private void setContent(Spanned content, Status.Mention[] mentions, List<Emoji> emojis,
                             StatusActionListener listener) {
         Spanned emojifiedText = CustomEmojiHelper.emojifyText(content, emojis, this.content);
-
         LinkHelper.setClickableText(this.content, emojifiedText, mentions, listener);
+        this.contentText = this.content.getText();
```

Comment:
Instead of saving the text as a member var I would just create it when needed.

Reply:
Well, to create `contentText` later, I need to save `content` or `emojifiedText`, `mentions` and `listener` somewhere (same for `mentionsOnlyContentText`).